gensim/utils.py

```diff
@@ -260,6 +260,9 @@ def load(cls, fname, mmap=None):
         is encountered.
 
         """
+        import warnings
+        warnings.warn("This method loads pickle dumps. If a corpora was saved with the save_corpus() method," + 
```

Comment:
The message, when emitted, will leave the user confused -- what method?

`corpora` is plural of `corpus`. Do not split messages in the middle with `\n`, leave formatting to the terminal. End sentences in full stop.